lib/success.js

```diff
@@ -56,6 +58,10 @@ export default async function success(pluginConfig, context, { Octokit }) {
 
   if (successComment === false) {
     logger.log("Skip commenting on issues and pull requests.");
+    logger.error(`Issue and pull request comments should be disabled via 'successCommentCondition'.
+      Using 'false' for 'successComment' is deprecated and will be removed in a future major version.`);
```

Comment:
use `logger.warn` and prefix with `DEPRECATION:`, something like

> `Setting successComment to false is deprecated, please use successCommentCondition instead

```suggestion
    logger.warn(`DEPRECATION: 'false' for 'successComment' is deprecated and will be removed in a future major version. Use 'successCommentCondition' instead.`);
```